### ISSUE TITLE
Issue: Latest Matched Price and Matched Total Not Populated on MarketOutcome Account

### ISSUE THREAD

**Issue by druspencer** [MonacoProtocol/protocol#209]
Following the release of `0.14.1` it has been observed that the fields `latestMatchedPrice` and `matchedTotal` are not populated on the `MarketOutcome` account.

![image](https://github.com/MonacoProtocol/protocol/assets/11299576/47cc38ac-bef9-4810-995f-9e61ee2dad7f)

As a workaround, to get the total traded amount on a market:
- Fetch all trades for a market
- Sum the stake of all the trade accounts
- Divide by 2 (as there are 2 trades for each matched bet)
 
If you wish to get it for an outcome, filter your trade query by `marketOutcomeIndex` once you have fetched the data from chain.

``` 
const example = async (marketPk: web3.PublicKey) => {
  const program = await getProgram();
  const trades = await Trades.tradeQuery(program)
    .filterByMarket(marketPk)
    .fetch();
  let tradesTotal = 0;
  for (const trade of trades.data.tradeAccounts) {
    tradesTotal += trade.account.stake.toNumber() / 10 ** 6;
  }
  // divide by 2 because each matched bet is 2 trades
  console.log("Total traded amount", tradesTotal / 2);
};

const marketPk = new web3.PublicKey(
  "DG6GEru4yZEcSY1gDxEKjCsopfbJ1o5cVfnGsjUr6vJY"
);
example(marketPk);

```

**Comment by pawel-betdex** [comment]
This potentially can be more efficient
```
const example = async (marketPk: web3.PublicKey, outcomeIndex: number) => {
  const program = await getProgram();
  const mmps = await MarketMatchingPools.marketMatchingPoolQuery(program)
    .filterByMarket(marketPk)
    .filterByMarketOutcomeIndex(outcomeIndex)
    .filterByForOutcome(true) // this ensures it's only a half
    .fetch();
  let totalMatchedAmount = 0;
  for (const mmp of mmps.data.marketMatchingPools) {
    totalMatchedAmount += mmp.account.matchedAmount.toNumber() / 10 ** 6;
  }
  console.log("Total matched amount", totalMatchedAmount);
};

const marketPk = new web3.PublicKey(
  "DG6GEru4yZEcSY1gDxEKjCsopfbJ1o5cVfnGsjUr6vJY"
);
example(marketPk);
```

**Comment by druspencer** [comment]
**Update**

Matched total can now be viewed on the liquidities account as of [0.14.2](https://github.com/MonacoProtocol/protocol/releases/tag/v0.14.2)

**Comment by eoin-betdex** [comment]
These fields were removed in #224 released in  [0.15.0](https://github.com/MonacoProtocol/protocol/releases/tag/v0.15.0)
